README.md

```diff
@@ -585,6 +585,16 @@ This is a grouping mechanism to group files together
 | WB    | Worm Base |
 | ZFIN  | Zebrafish Information Network |
 
+### Including corresponding LinkML version in the JSON file submission header
+
+The LinkML version for which the file is being submitted now needs to be added to the JSON file header, for example:
+
+```json
+{
+  "linkml_version" : "v1.3.2",
```

Comment:
@markquintontulloch just wondering that if we could maybe look for this but if it's null then just assume the current version?

Reply:
yeah, Christian mentioned that - I'm in two minds about it.  It think just sticking the current version on there may cause more issues than it solves in the long term.  Better error handling is definitely a good idea though.